lib/Controllers/list.js

```diff
@@ -68,6 +69,14 @@ List.prototype.fetch = function(req, res, context) {
         order.push([sortColumn, 'ASC']);
       }
     });
+    var allowedColumns = this.resource.sort.attributes || Object.keys(model.rawAttributes);
+    var columnNames = order.map(_.first);
```

Comment:
this might be a premature optimization, but if we're already building the order array above perhaps you could build the columnNames at the same time, thus reducing the need for this mapping